packages/design-system/src/styles/base/_typography.scss

```diff
@@ -132,7 +121,9 @@
 .ds-text--lead,
 .ds-text-body--lg,
 .ds-text-body--md {
-  line-height: $base-line-height;
+  color: $typography-body__color;
```

Comment:
Loki also picks up a visual difference in the drawer footer content that uses the `ds-text` component. This new `color` declaration changes the color of the text in that component from a color set by a `dialog` rule to `#404040`

Reply:
Hrm, it looks like there isn't currently a variable for this, so would need to create something like `drawer-footer__color` to add to the drawer component theme and set it in `_Drawer.scss`, sound like the right thing?

Reply:
I'm wondering if we need to actually remove the color declaration here because these are supposed to inherit their color from `ds-base`. If this came from a mgov override, maybe it actually didn't need to be there. Was it [this one?](https://github.com/CMSgov/design-system/pull/2079/files/63a9a7c002ceeccd7edf387911be24622d94ca2e#diff-6b1d7de20b3bbfaf673aa2826fc6b181293c8de6cb898e66001afc9e499cbbf9L95)
```css
color: $black-500;
```
What if we removed it entirely? Does the `ds-text-body` need a different font color, @babsdenney ?

Reply:
On the other hand, the styles we're getting from that `dialog` rule are probably wrong to begin with. In that case, the snapshot change we're seeing is correct, and we might even want to remove that `.dialog` color declaration from core.

Reply:
It looks like `black-500` doesn't even exist, it was removed at some point with tokens. The medicare references file needs to be updated with granite values that reflect the old black values.

Reply:
Scratch my last deleted comment.. `black-500` is mapped to `granite-800` in tokens which is the same as `base` for medicare.  Since that was an override, I can't see how it would have changed other than inheriting from dialog like you noted which is why I suggested a custom token and color setting for drawer footer..  At this point I'm a little confused :)

Reply:
That sounds good! I like the option of removing it entirely so it inherits the ds-base style. Headers and body styles use the same color ds-base color.

Reply:
I've removed this component style so we inherit `ds-base` which is the lighter grey color for body items which passes over into list headings and such. I've updated loki snapshots which you can both look at to verify these changes look correct? 

Also there is an update to the drawer background for the close button in core which I believe is now the standard 'outline' style which does not have a background (was previously white)

Reply:
Those changes look good to me, but @babsdenney or @line47 should give the final word

Reply:
Looks good to me as well :+1:

Reply:
I'll give @babsdenney the final review